lib/ddtrace/contrib/resque/resque_job.rb

```diff
@@ -30,6 +26,10 @@ def after_perform(*args)
   pin = Datadog::Pin.get_from(Resque)
   next unless pin && pin.tracer
   pin.tracer.set_service_info(pin.service, 'resque', Datadog::Ext::AppTypes::WORKER)
+
+  # Create SyncWriter instance before forking
+  sync_writer = Datadog::SyncWriter.new(transport: pin.tracer.writer.transport)
+  Datadog::SyncWriter::CACHED_INSTANCE = sync_writer
```

Comment:
maybe we can move `CACHED_INSTANCE` under the Resque namespace instead of the `SyncWriter`